tdrs-backend/tdpservice/search_indexes/admin/tanf.py

```diff
@@ -109,12 +109,14 @@ class TANF_T7Admin(admin.ModelAdmin):
     """ModelAdmin class for parsed T7 data files."""
 
     list_display = [
-        'record',
-        'rpt_month_year',
+        'RecordType',
```

Comment:
i think here we need basically all the data. it is pretty challenging to check the transformation currently. need stratum, tdrs indicator, and families.

 and probably the ability to filter based on TANF vs SSP. and STT but could be a future ticket.

Reply:
I added additional fields to the `list_display`, but i agree the filtering should be looked at in another ticket if possible. let me know if anything else would be helpful